site/src/_layoutsPlayground/component.hbs

```diff
@@ -0,0 +1,45 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <title>{{default title 'd3fc'}}</title>
+    <link href="https://cdnjs.cloudflare.com/ajax/libs/d3fc/{{ package.version }}/d3fc.min.css" rel="stylesheet" />
+</head>
+
+<body>
+    <svg id="chart" width="100%" height="300px"></svg>
+
+    <!-- <script src="https://cdnjs.cloudflare.com/ajax/libs/d3/{{ slice package.dependencies.d3 1 }}/d3.js"></script> -->
+    <!-- <script src="https://cdnjs.cloudflare.com/ajax/libs/css-layout/{{ slice package.dependencies.css-layout 1 }}/css-layout.js"></script> -->
+    <script src="https://cdnjs.cloudflare.com/ajax/libs/d3fc/{{ package.version }}/d3fc.bundle.min.js"></script>
+    <script data-src="{{basename}}.js">
+var width = document.getElementById('chart').getBoundingClientRect().width || 600,
```

Comment:
didn't know that function and was just changing the createFixture to something clearer. Will switch